### PR TITLE
[express-http-proxy]: fix typings for proxyRes, proxyReq

### PR DESCRIPTION
**Pull request by RobinCsl**
TL;DR: `proxyRes` and `proxyReq` come from calling `request` from the `http` or `https` Node.js libraries.

The request is made in [`sendProxyRequest`](https://github.com/villadora/express-http-proxy/blob/master/app/steps/sendProxyRequest.js#L12-L13), and `requestModule` is set in [`resolveProxyHost`](https://github.com/villadora/express-http-proxy/blob/master/app/steps/resolveProxyHost.js#L15) through the [`parseHost`](https://github.com/villadora/express-http-proxy/blob/2aec18894bfe4d06fc61d959cf718c9e1496226e/lib/requestOptions.js#L23) function, which sets the `module` property to the `http` or `https` Node.js modules.

### CODE REVIEW

**Review by carboneater — APPROVED** [review]
Good find!

**Comment by RobinCsl** [comment]
Thank you for your review, Gabriel!
